### ISSUE TITLE
FSharpLint not working

### ISSUE THREAD

**Issue by quantum-booty** [ionide/Ionide-vim#52]
I've installed FSharpLint by doing ```dotnet tool install -g dotnet-fsharplint```, and tried one of their examples on the [FSharpLint website](https://fsprojects.github.io/FSharpLint/):
```
type ExampleInterface =
   abstract member print : unit -> unit

[<EntryPoint>]
let main argv =
    let x = List.fold (fun x y -> x + y) 0 [1;2;3]
    printfn "%d" x
    0
```

**Expected behaviour**
I expect that Nvim should give me virtual text diagnostic warnings that are like this:
![image](https://user-images.githubusercontent.com/28958399/158018879-68928489-142b-4554-b53f-34b473c56258.png)
however I am not seeing any warnings:
![image](https://user-images.githubusercontent.com/28958399/158018907-bdb1fbb4-af6c-45ca-a4f8-c6ff1bb1fef5.png)

**Environment (please complete the following information):**
 - OS: Archlinux
 - Vim / Neovim version: NVIM v0.7.0-dev+1241-g08d9d74fd 
 - dotnet SDK version: 6.0.102
 - mono / .Net Framework version: 6.0.102

**Comment by cannorin** [comment]
FSharpLint seems to be disabled in FSAC's side: https://github.com/fsharp/FsAutoComplete/blob/main/src/FsAutoComplete.Core/Commands.fs#L1277-L1302

**Comment by baronfel** [comment]
yeah, we have an open item to look at reenabling it - the core problem is one of FCS incompatibility. We either need to help that project stay up to date on FCS, have a stable FCS build, or find a way to invoke it in a way that lets that tool load its own FCS.
